components/ProfilePage/FollowButton.tsx

```diff
@@ -107,10 +108,17 @@ export const FollowButton = ({
     <Col className={`d-flex w-100 justify-content-start`}>
       <div>
         <div className="view-edit-profile">
-          <Button onClick={handleClick} className={`btn btn-lg py-1`}>
+          <FillButton
+            onClick={handleClick}
+            className={`py-1`}
+            Icon={checkmark}
+            label={text}
+          />
+
+          {/* <Button onClick={handleClick} className={`py-1`}>
```

Comment:
the refactored FillButton seems to work great!  Can probably ditch the older commented out code, perhaps?